src/analyze.ts

```diff
@@ -453,6 +472,12 @@ export default async function analyze(id: string, code: string, job: Job): Promi
           }
         }
       }
+      else if (job.analysis.computeFileReferences && node.type === 'MemberExpression' && node.object.type === 'MetaProperty' && node.object.meta.name === 'import' && node.object.property.name === 'meta' && (node.property.computed ? node.property.value : node.property.name) === 'url') {
+        // import.meta.url leaf trigger
+        staticChildValue = { value: importMetaUrl };
+        staticChildNode = node;
+        backtrack(parent!, this);
```

Comment:
The `!` seems like an error.

We should probably have `&& parent` in the `if` condition above if the parent is possibly null or undefined.